lms/envs/common.py

```diff
@@ -1747,6 +1747,16 @@
     'draggabilly': 'js/vendor/draggabilly.js'
 }
 
+########################## DJANGO WEBPACK LOADER ##############################
+
+WEBPACK_LOADER = {
+    'DEFAULT': {
+        'BUNDLE_DIR_NAME': 'bundles/',
```

Comment:
This is relative to `common/static`, so the final path of bundles will be e.g., `common/static/bundles/CourseOutline-hash.js`